### DISCUSSION TITLE
MAUI BUG is more than feature

### DISCUSSION THREAD

**Discussion by 3egirlsdream** [dotnet/runtime#95674] [General]
lots of bug exsist many years ,  and remain are not repaired, what are u doing ?   those developer . do more on this framework, not coffee. otherwise, throw it into the trash can while you can.

I'm very disappointed.

**Comment by GeraudFabien** [comment]
Maui is different than runtime. You are not in the right repo
Of you have issue use blazor, uno plate-form, avalonia or any other technologies in other language like react native, flutter,...

**Comment by AraHaan** [comment]
If you have issue, make your own in .NET. Nothing is stopping you from making your own implementation for MAUI 😂. The source code is legit there for you to fork and make changes how you see fit.

**Comment by lloydjatkinson** [comment]
> I'm very disappointed.

![image](https://github.com/dotnet/runtime/assets/5633938/2931775a-8011-4a49-bd7a-c231bcdc805c)
